packages/playwright-core/src/server/injected/injectedScript.ts

```diff
@@ -1130,10 +1130,45 @@ export class InjectedScript {
       this.hideHighlight();
     this._highlight = new Highlight(this);
     this._highlight.install();
+
     const elements = [];
     for (const selector of selectors)
       elements.push(this.querySelectorAll(selector, this.document.documentElement));
+    const elementsToMask = this.document.querySelectorAll('[data-test-screenshot="mask"]') as NodeListOf<HTMLElement>;
```

Comment:
Let's use `this.querySelectorAll()` to work inside shadow dom.

Reply:
Doh, that requires parsed selector...